### PR TITLE
Update marketplace.md

### PR DESCRIPTION
**Pull request by aashishkohli**
updates to MP docs to cover https://github.com/ClickHouse/clickhouse-docs/issues/2075 and other cosmetic changes

### CODE REVIEW

File: docs/en/cloud/marketplace.md [hunk 1]
```diff
@@ -33,37 +34,39 @@ title: CSP Marketplace
...
 
 You can switch between organizations in the top right corner of the ClickHouse Cloud console.
 
-## I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
+### I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
 
-Please contact [ClickHouse Cloud support](https://clickhouse.cloud/support) in this case.
+Please follow the steps outlined above to subscribe and sign up via the marketplace. A new organization will be created for you. After this, contact [ClickHouse Cloud support](https://clickhouse.cloud/support) and share the organization ID for the old organization (the one which has existing services and workloads), and the new org (that you created via the marketplace). You will also need to share the email for a user that has the admin role on both orgs.  ClickHouse Cloud support team will then be able to connect your existing services to the new org so that your usage starts getting billed via the marketplace.
```
Comment (minodudd) [inline docs/en/cloud/marketplace.md:53]: This is incorrect.
The information collected here:
* Org with MP subscription and without services (source org)
* Org with services but no MP subscription (target org)
* Admin user who exists on both

Is used to move the MP subscription from the source org to the target org. We don't move the services as implied by:
> ClickHouse Cloud support team will then be able to connect your existing services to the new org

File: docs/en/cloud/marketplace.md [hunk 2]
```diff
@@ -33,37 +34,39 @@ title: CSP Marketplace
...
 
-## I subscribed as a marketplace user and now I want to unsubscribe from the ClickHouse Cloud.
+### I subscribed as a marketplace user and now I want to unsubscribe from ClickHouse Cloud.
 
 Note that you can simply stop using ClickHouse Cloud and delete all existing ClickHouse Cloud services. Even though the subscription will still be active, you will not be paying anything as ClickHouse Cloud doesn't have any recurring fees.
 
-If you want to unsubscribe, please navigate to the Cloud Provider console and cancel the subscription renewal there. Once the subscription ends, all existing services will be stopped and you will be prompted to add a credit card. If no card was added, after two weeks all existing services will be deleted.
+If you want to unsubscribe, please navigate to the Cloud Provider console and cancel the subscription renewal there. Once the subscription ends, all existing services will be stopped and you will be prompted to add a credit card. If no card is added, after two weeks all existing services will be deleted.
```
Comment (minodudd) [inline docs/en/cloud/marketplace.md:59]: We've matched this flow with the credit card flow:
Cancelling the MP subscription will start the clock on the following actions:
* Once we get the subscription cancellation notice from Tackle the org can no longer create new services or start stopped services
* 14 days after the cancellation notice from tackle we stop the services (they can't be started by the user)
* 30 days after the cancellation notice from tackle we decommission the organization, which will include deleting all services and their data.

File: docs/en/cloud/marketplace.md (hunk 1 above)
Comment (aashishkohli) [reply docs/en/cloud/marketplace.md:53]: thanks @minodudd - that is what I meant and I understand that is the workflow, let me reword it to clarify.

File: docs/en/cloud/marketplace.md [hunk 3]
```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
 title: CSP Marketplace
 ---
 
-## How can I start using ClickHouse Cloud connected to my AWS/GCP/Azure account billing?
+## Sign up for ClickHouse Cloud connected to your Cloud Service Provider (CSP) - AWS/GCP/Azure - billing account
```
Comment (johnnymatthews) [inline docs/en/cloud/marketplace.md:7]: This is a very long title. Can we turn it into a paragraph instead?

```suggestion
This page covers how to sign up for ClickHouse Cloud, that is connected to a cloud service provider, such as AWS, GCP, or Azure.
```

We'll also have to update the header levels to `##` if we're changing this title.

File: docs/en/cloud/marketplace.md [hunk 4]
```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
...
 
 ### AWS 
 
-- Log into the AWS console using your AWS account
-- Navigate to the [ClickHouse Cloud at aws marketplace](https://aws.amazon.com/marketplace/pp/prodview-jettukeanwrfc)
+- Log into the AWS console using your AWS account. This is the account you intend to subscribe to the ClickHouse Cloud service.
+- Navigate to [ClickHouse Cloud on AWS marketplace](https://aws.amazon.com/marketplace/pp/prodview-jettukeanwrfc) product listing
```
Comment (johnnymatthews) [inline docs/en/cloud/marketplace.md:12]:
```suggestion
1. Log into the AWS console using your AWS account. This is the account you intend to subscribe to the ClickHouse Cloud service.
1. Navigate to [ClickHouse Cloud on AWS marketplace](https://aws.amazon.com/marketplace/pp/prodview-jettukeanwrfc) product listing.
```

File: docs/en/cloud/marketplace.md [hunk 5]
```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
...
 - Click "View purchase options"
 - In the "Contract Options" section of the page, enter any number in the Units field. This will not affect the price your pay as the price for these units for the public offering is $0. These units are usually used when accepting a private offer from ClickHouse Cloud.
```
Comment (johnnymatthews) [inline docs/en/cloud/marketplace.md:14]:
```suggestion
1. Click **View purchase options**.
1. In the **Contract Options** section of the page, enter any number in the **Units** field. This will not affect the price your pay as the price for these units for the public offering is $0. These units are usually used when accepting a private offer from ClickHouse Cloud.
```

File: docs/en/cloud/marketplace.md [hunk 6]
```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
...
-  - If you are a new CH Cloud user, click "Register" at the bottom of the page. You will be prompted to create a new user and verify the email. After verifying your email, you can leave the ClickHouse Cloud login page and login using the new username at the https://clickhouse.cloud.
-  - If you are an existing CH Cloud user, simply log in using your credentials.
+- Click "Create contract", and "Pay now" on the subsequent screen
+- Click "Set up your account"
+- You will be redirected to the AWS marketplace ClickHouse Cloud login page. Please, complete your sign-in / sign-up at this page so we can bind your ClickHouse Cloud organization to AWS billing:
+  - If you are a new ClickHouse Cloud user, click "Register" at the bottom of the page. You will be prompted to create a new user and verify the email. After verifying your email, you can leave the ClickHouse Cloud login page and login using the new username at  https://clickhouse.cloud
+  - If you are an existing ClickHouse Cloud user, simply log in using your credentials
```
Comment (johnnymatthews) [inline docs/en/cloud/marketplace.md:19]:
```suggestion
1. Click **Create contract**, and then **Pay now**.
1. Click **Set up your account**.
1. You will be redirected to the AWS marketplace ClickHouse Cloud login page. Please, sign-in or sign-up at this page. Doing this binds your ClickHouse Cloud organization to AWS billing:
  a. If you are a new ClickHouse Cloud user, click **Register** at the bottom of the page. You will be prompted to create a new user and verify the email. After verifying your email, you can leave the ClickHouse Cloud login page and login using the new username at [clickhouse.cloud](https://clickhouse.cloud).
  a. If you are an existing ClickHouse Cloud user, simply log in using your credentials.
```

File: docs/en/cloud/marketplace.md [hunk 7]
```diff
@@ -4,20 +4,21 @@ slug: /en/cloud/marketplace
...
 - After successful log in, a new ClickHouse Cloud organization will be created. This organization will be connected to your AWS billing account.
+- NOTE: If you are an existing ClickHouse Cloud user and you would like your current usage to be billed via the marketplace please follow steps outlined below in ("I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace").
```
Comment (johnnymatthews) [inline docs/en/cloud/marketplace.md:21]:
```suggestion
1. After successfully logging in, a new ClickHouse Cloud organization will be created. This organization will be connected to your AWS billing account.

:::note
If you are an existing ClickHouse Cloud user and you would like your current usage to be billed using the marketplace, please follow steps outlined below in the [I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace](#i-am-an-existing-clickhouse-cloud-user-and-i-want-my-existing-services-to-be-billed-via-marketplace).
:::
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Request some minor formatting changes. Content is good!
